lib/autocomplete-manager.js

```diff
@@ -138,16 +142,22 @@ export default class AutocompleteManager {
     }
   }
 
+  watchEditor (editor, labels) {
```

Comment:
Is this the public method you plan on having other packages consume? If so, could you add some basic tests as well as documentation to the readme about the service and this method?